### PR TITLE
REALM-7

### PR DESCRIPTION
**Pull request by paugajt**
Updated the UI for the event detail page

### CODE REVIEW

File: helpers/networking.js [hunk 1]
```diff
@@ -1,9 +1,13 @@
...
 import {logUserIn} from './userHelper';
 
+const devServer = "https://otq-dev.herokuapp.com";
+const prodServer = "https://onthequad.herokuapp.com";
+const server = prodServer;
+
 export async function searchEvents(searchString) {
    let eventArray = []
-   let url = 'https://onthequad.herokuapp.com/events?search=' + searchString
+   let url = server + '/events?search=' + searchString
```
Comment (daniellund14) [inline helpers/networking.js]: This is better but we probably want to add an env story

File: screens/EventDetailScreen.js [hunk 2]
```diff
@@ -47,42 +51,67 @@ class EventDetailScreen extends Component {
...
+ formatTime(time) {
+  let hours = Number(time.slice(0,2))
+  let minutes = Number(time.slice(3,5))
+  let timeOfDay = hours > 12 ? 'PM' : 'AM';
+
+  if (hours > 12) { hours = hours - 12}
+  if (hours == 0) { hours = 12 }
+  if (minutes < 10) { minutes = '0' + minutes}
+
+  return hours + ':' + minutes + " " + timeOfDay
```
Comment (daniellund14) [inline screens/EventDetailScreen.js]: There might be a way to go from a utc string to better formatted datetime. I’m betting there is a library

File: helpers/networking.js (hunk 1 above)
Comment (paugajt) [reply helpers/networking.js]: Yeah we can add that to the backlog

File: screens/EventDetailScreen.js (hunk 2 above)
Comment (paugajt) [reply screens/EventDetailScreen.js]: javascript has a built in way to transform utc strings to dates but for some reason it gave me a lot of weird errors when trying to use it. This was my workaround

Comment (daniellund14) [reply screens/EventDetailScreen.js]: Workarounds get scary if we keep adding them, is it worth doing a follow up to clean this up, or should we try and fix it with this release?

Comment (paugajt) [reply screens/EventDetailScreen.js]: I can work on cleaning it up tonight and if it takes longer than an hour then I'll create a follow up story

Comment (daniellund14) [reply screens/EventDetailScreen.js]: 👍

Comment (paugajt) [reply screens/EventDetailScreen.js]: Looking at stack overflow and google, it doesn't look like there is a built in way to do this. Every post suggests to do something similar to what I'm doing here

File: screens/EventDetailScreen.js [hunk 3]
```diff
@@ -98,6 +98,7 @@ class EventDetailScreen extends Component {
                       title='Sign Up'
                       onPress={() => console.log('Sign up button pressed')}
                     />
+                    <Text>{this.state.spotsAvailable} spots left</Text>
```
[the remark this answers is not in the document]
Comment (paugajt) [reply screens/EventDetailScreen.js]: There's a calculated one that I should have been using here. Its updated now

**Pull request merged**